### PR TITLE
[TE] frontend - Ttbach/created custom templates

### PR DESCRIPTION
**Pull request by ttbach**
- Added the following custom templates for the anomaly-table: start-duration, current-wow, investigation-link
- Moved anomalies mock data to mirage
- Replaced stub code in template with table
- Note: CSS will be fixed in later PRs
screen shot 2018-02-28 at 4 14 30 pm

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<p class="anomalies-table__values">
+  {{record.current}}/{{record.baseline}}
+</p>
+<p class="anomalies-table__change anomalies-table__change--{{color-delta record.change}}">
+  {{record.change}}
+</p>
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs]: you know the drill ;-)

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<p class="anomalies-table__values">
+  {{record.current}}/{{record.baseline}}
+</p>
+<p class="anomalies-table__change anomalies-table__change--{{color-delta record.change}}">
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs]: screenshot didn't show the colors. Is this working?

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/investigation-link/template.hbs [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{{#link-to 'rootcause' (query-params anomalyId=record.id)
+  class="te-anomaly-table__link"}}
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/investigation-link/template.hbs]: we're already have styling for `thirdeye-link`, if this is slightly different, how about adding a modifier like `thirdeye-link--large`

File: thirdeye/thirdeye-frontend/app/pods/home/route.js [hunk 4]
```diff
@@ -39,7 +39,7 @@ export default Route.extend({
 
       anomalyMapping[metric][functionName].push(anomaly);
       anomaly.value = value;
-      anomaly.investigationLink = `/rootcause?anomalyId=${anomaly.id}`;
+      anomaly.change = ((current-baseline)/baseline)*100;
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/home/route.js]: we might want to add a `.toFixed(2)` here

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<p class="anomalies-table__values">
+  {{record.current}}/{{record.baseline}}
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs]: are those numbers rounded? if not, perhaps use the d3 formatter? 
(some metrics have values in the millions+, and we don't want the text to overflow) 
https://github.com/linkedin/pinot/blob/master/thirdeye/thirdeye-frontend/app/utils/utils.js#L52

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs (hunk 2 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs]: @justYves Yeah, a lot of these classes are stubs, so I can figure them out later.

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/investigation-link/template.hbs (hunk 3 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/investigation-link/template.hbs]: @justYves This is temporary. I'm planning on trying to incorporate Steve's `te-anomaly-table` styles, which contain styles for this.

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs (hunk 5 above)
Comment (ttbach) [reply thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs]: Good call. Thanks.

File: thirdeye/thirdeye-frontend/app/pods/home/route.js [hunk 6]
```diff
@@ -37,9 +34,13 @@ export default Route.extend({
...
 
+      // Group anomalies by metric and function name
       anomalyMapping[metric][functionName].push(anomaly);
-      anomaly.value = value;
-      anomaly.investigationLink = `/rootcause?anomalyId=${anomaly.id}`;
+
+      // Format current and baseline numbers, so numbers in the millions+ don't overflow
+      anomaly.current = humanizeFloat(anomaly.current);
+      anomaly.baseline = humanizeFloat(anomaly.baseline);
+      anomaly.change = (((current-baseline)/baseline)*100).toFixed(2);
```
Comment (newsummit) [inline thirdeye/thirdeye-frontend/app/pods/home/route.js]: readability enhancer... a little spacing would be good here, around the `-`

File: thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs (hunk 5 above)
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/current-wow/template.hbs]: Yeah, I had issues with unusually long numbers here. Fixed them to 2 decimal places.

**Pull request merged**